torch/_inductor/codegen/cpp.py

```diff
@@ -3486,6 +3374,41 @@ def _select_tiling_indices(
             return contig_vars_sorted
         return sorted(contig_vars_sorted, key=contig_vars_list.count)[-1:]
 
+    @staticmethod
+    def _get_dtype_from_loopbodies(loop_bodies):
```

Comment:
How about moving it to cpp_utils.py, considering it is used by other classes too?

Reply:
Updated, thanks for the suggestion.